### ISSUE TITLE
Remove duplicate tutorial files

### ISSUE THREAD

**Issue by MDeiml** [tequilahub/unitaria#122]
The notebook tutorials are now duplicate, once in the `unitaria-tutorials` folder and once in the `examples` folder. The latter one should be deleted
